### PR TITLE
Dart 2.17

### PR DESCRIPTION
**Pull request by johnpryan**

### CODE REVIEW

File: command_line/mono_pkg.yaml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 sdk:
-  - dev
-  - stable
```
Comment (domesticmouse) [inline command_line/mono_pkg.yaml:3]: are you sure you want to lose stable?

**Review by domesticmouse — APPROVED** [review]

File: command_line/mono_pkg.yaml (hunk 1 above)
Comment (johnpryan) [reply command_line/mono_pkg.yaml:3]: Planning to add this back after the release

File: server/simple/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Official Dart image: https://hub.docker.com/_/dart
 # Specify the Dart SDK base image version using dart:<version> (ex: dart:2.12)
-FROM dart:stable AS build
+FROM dart:beta AS build
```
Comment (parlough) [inline server/simple/Dockerfile:3]: Can this go back to `stable`?

```suggestion
FROM dart:stable AS build
```

Comment (johnpryan) [reply server/simple/Dockerfile:3]: 👍 good catch!

**Review by parlough — APPROVED** [review]

File: server/simple/Dockerfile (hunk 2 above)
Comment (parlough) [reply server/simple/Dockerfile]: Actually, I'm sorry. I guess the docker images haven't been updated yet? I thought https://github.com/dart-lang/dart-docker/commit/a6701292b23f948e0c5ddb185d0d18812cf5501f would update it but maybe it's a manual process.

Comment (johnpryan) [reply server/simple/Dockerfile]: It's fine, it will be updated soon!

**Pull request merged**